Automattic/wp-calypso
# Square Payments: Create landing page

Related to https://github.com/Automattic/dotcom-forge/issues/7781
Related to https://github.com/Automattic/dotcom-forge/issues/7785

## Proposed Changes

* Create a landing page to be shown on the `/start-with/square-payments`
* Update page layout to mach the design in figma
* Add a CTA to redirect to the store creation endpoint 
* Send a track event for page view and CTA click

PS: Mobile visualization will be handled in a follow-up PR

## Why are these changes being made?

As part of the Partner Aware Onboarding project to shows users a page to about the partnership before starting a store with those definitions selected.

## Testing Instructions

* Go to `/start-with/square-payments` in a logged in and logged out session
* Check if the layout matches the pictures below
* Click on the CTA and check if you are redirected to `/setup/entrepreneur/start?partnerBundle=square`
* Check on tracks if the events are being properly recorded

| Logged out  | Logged in |
| ------------- | ------------- |
|![CleanShot 2024-06-20 at 11 00 34@2x](https://github.com/Automattic/wp-calypso/assets/5039531/084f2491-cc18-4e34-bc79-c7408a7601b2)|![CleanShot 2024-06-20 at 11 02 06@2x](https://github.com/Automattic/wp-calypso/assets/5039531/323a9d7c-2170-4658-87f3-9ee9c2acdf32)|
